slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -106,10 +106,26 @@ void SLAMServiceImpl::SetUpMapBuilder() {
     mapBuilder.BuildMapBuilder();
 }
 
-void SLAMServiceImpl::CreateMap() {
+void SLAMServiceImpl::ProcessDataOffline() {
```

Comment:
`CreateMap` was for the most part already `ProcessDataOffline`. `ProcessDataOffline` will require more work with future tickets, but should satisfy the needs for this ticket for now.